source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -690,6 +690,27 @@ private void MatchProperties(PimsAcquisitionFile acquisitionFile, IEnumerable<Us
                         acquisitionProperty.Property = _propertyService.PopulateNewProperty(acquisitionProperty.Property);
                     }
                 }
+                else if(!string.IsNullOrWhiteSpace(acquisitionProperty.Property.SurveyPlanNumber))
```

Comment:
Ok, do we know if this is just an acquisition - specific requirement? What happens if I try and add a common property to another file type?

Reply:
For now is only in acquisition; I presume all others might need it, but for now the common properties are only used by the business for acquisition files.